frontend/tests/controllers/ProblemCreateTest.php

```diff
@@ -829,6 +829,83 @@ public function testValidProblemInteractive() {
         );
     }
 
+    /**
+     * Test that we are able to submit a lecture problem without public tags.
+     */
+    public function testValidLecture() {
+        // Get the problem data
+        $problemData = \OmegaUp\Test\Factories\Problem::getRequest(new \OmegaUp\Test\Factories\ProblemParams([
+            'zipName' => OMEGAUP_TEST_RESOURCES_ROOT . 'readonlylecture.zip',
+            'languages' => '',
+            'selectedTags' => '',
+        ]));
+        $r = $problemData['request'];
+        $problemAuthor = $problemData['author'];
+
+        // Login user
+        $login = self::login($problemAuthor);
+        $r['auth_token'] = $login->auth_token;
+
+        // Call the API
+        $response = \OmegaUp\Controllers\Problem::apiCreate($r);
+
+        // Validate
+        // Verify response
+        $this->assertSame('ok', $response['status']);
+
+        // Verify data in DB
+        $problems = \OmegaUp\DAO\Problems::getByTitle($r['title']);
+
+        // Check that we only retreived 1 element
+        $this->assertSame(1, count($problems));
```

Comment:
Ya existe una función para contar elementos de un array ([`assertCount`](https://docs.phpunit.de/en/9.6/assertions.html#assertcount)), así que esto puede quedar así:
```suggestion
        $this->assertCount(1, $problems);
```